### DISCUSSION TITLE
Tearing mode locks the cursor fps to the game's fps.

### DISCUSSION THREAD

**Discussion by Sinaaaa** [hyprwm/Hyprland#4560] [Q&A] [answered]
Whenever I enable tearing in my game that I play with a frame limiter of either 30 or 40 fps, the cursor starts operating at that framerate. I'm more than happy to play locked at 30fps with tearing, but the cursor lag is unbearable.

Is there any way to fix this?

**Comment by vaxerski** [comment, marked as answer]
yes we have to change the code
